edgemesh/pkg/server/tcp.go

```diff
@@ -15,12 +15,15 @@ import (
 )
 
 func StartTCP() {
-	server := config.GetString("server", "0.0.0.0")
-	port := config.GetString("port", "8080")
-
-	log.LOGGER.Infof("start listening at %s:%s", server, port)
 
-	listener, err := net.Listen("tcp", server+":"+port)
+	server, err:= getIP()
+	if err != nil {
+		log.LOGGER.Errorf("TCP server start error : %s", err)
```

Comment:
```startDnsServer``` return when got error, ```StartTCP``` should also return?

if return ip is nil, will got panic

Reply:
+1